ocs_ci/framework/conf/default_config.yaml

```diff
@@ -21,6 +21,7 @@ RUN:
   cli_params: {}  # this will be filled with CLI parameters data
   client_version: '4.1.4'
   bin_dir: './bin'
+  google_api_secret: '~/.ocs-ci/google_api_secret.json'
```

Comment:
Do we have a doc that states what should we have in this file

Reply:
We will.
For now, it will skip in case this file is missing

Reply:
Shouldn't we just skip the uploading if the file is missing, and still perform the tests?

Reply:
We actually thought about it and there's no point in running performance test if you don't have the data to analyze so skip will save execution time